frontend/src/employee-frontend/components/unit/unit-reservations/ChildDay.tsx

```diff
@@ -51,34 +94,36 @@ export default React.memo(function ChildDay({ day, childReservations }: Props) {
       ? getTimesOnWeekday(serviceTimes, day.date.getIsoDayOfWeek())
       : null
 
+  const expectedTimeForThisDay = getReservationOrServiceTimeOfDay(
+    dailyData.reservations,
+    serviceTimeOfDay
+  )
+  const maybeAsterisk = expectedTimeForThisDay?.type === 'service-time' && '*'
```

Comment:
I think the current spec uses ` (s)` instead of an asterisk.

Reply:
Using ` (s)` introduced some problems with our fixed with container. Circumvented the problem by shrinking the rightmost action column and preventing text wrapping.